addOns/llm/src/main/resources/org/zaproxy/addon/llm/resources/Messages.properties

```diff
@@ -1,4 +1,6 @@
-llm.desc = ZAP LLM Add-On
+llm.name = LLM Extension
```

Comment:
Why Extension here? (vs Add-on or Support)

Reply:
Because this is an extension but I can drop that.